scripts/preprocess_data.py

```diff
@@ -71,14 +71,14 @@ def main():
     
     text = tokenized_train_examples + tokenized_dev_examples
     
-    count_vectorizer.fit(text)
+    count_vectorizer.fit(tqdm(text))
 
-    vectorized_train_examples = count_vectorizer.transform(tokenized_train_examples)
-    vectorized_dev_examples = count_vectorizer.transform(tokenized_dev_examples)
+    vectorized_train_examples = count_vectorizer.transform(tqdm(tokenized_train_examples))
+    vectorized_dev_examples = count_vectorizer.transform(tqdm(tokenized_dev_examples))
 
-    dev_count_vectorizer = CountVectorizer(stop_words='english', max_features=args.vocab_size, token_pattern=r'\b[^\d\W]{3,30}\b')
-    reference_matrix = dev_count_vectorizer.fit_transform(tokenized_dev_examples)
-    reference_vocabulary = dev_count_vectorizer.get_feature_names()
+    # dev_count_vectorizer = CountVectorizer(stop_words='english', max_features=args.vocab_size, token_pattern=r'\b[^\d\W]{3,30}\b')
```

Comment:
If these lines are no longer used, we should delete them.

Reply:
I actually made an update to consolidate the `preprocess_data` and `make_reference_corpus` scripts. By default, `preprocess_data` will make the reference corpus from the validation data. But you can supply an additional arg if you'd like to make a custom reference corpus. See newest commits!